packages/back-end/src/integrations/SqlIntegration.ts

```diff
@@ -60,7 +63,6 @@ import {
   DropTableQueryParams,
 } from "../types/Integration";
 import { DimensionInterface } from "../../types/dimension";
-import { IMPORT_LIMIT_DAYS } from "../util/secrets";
```

Comment:
Does this need to be removed? I don't see it being deleted.

Reply:
It gets deleted in the `getTestQuery()` method below.